src/05-objects.js

```diff
@@ -132,7 +168,11 @@ function addKeyValuePairToObject(object, key, val) {}
  *  deleteKeyFromObject({ left: true, right: false }, "left");
  *  //> { right: false }
  */
-function deleteKeyFromObject(object, key) {}
+function deleteKeyFromObject(object, key) {
+  delete object[key];
+
```

Comment:
Let's try to keep an eye on these empty lines moving forward.  We need to be very deliberate in our code where we decide to place empty lines and they should help readability